task/aws/resources/data_source_image.go

```diff
@@ -34,7 +34,7 @@ func (i *Image) Read(ctx context.Context) error {
 	image := i.Identifier
 	images := map[string]string{
 		"ubuntu": "ubuntu@099720109477:x86_64:*ubuntu/images/hvm-ssd/ubuntu-focal-20.04*",
-		"nvidia": "ubuntu@898082745236:x86_64:Deep Learning AMI GPU CUDA 11.3.1 (Ubuntu 20.04) 20220303",
+		"nvidia": "ubuntu@898082745236:x86_64:Deep Learning AMI GPU CUDA 11.3.1 (Ubuntu 20.04) *",
```

Comment:
while we're at it, afaik `11.3.x` are not meant to break compatibility.

```suggestion
		"nvidia": "ubuntu@898082745236:x86_64:Deep Learning AMI GPU CUDA 11.3.* (Ubuntu 20.04) *",
```